Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+function identify_timepoints
+{
+    local subject=$1
+    local tplist=""
+    local tp visit n
+
+    #build the list of timepoints (sessions)
+    n=0
+    for session in ${PossibleVisits[*]}; do
+        tp="${subject}_${session}"
+        if [ -d "$StudyFolder/$tp" ] && ! [[ " ${ExcludeVisits[*]} " =~ [[:space:]]"$tp"[[:space:]] ]]; then
+             if (( n==0 )); then 
+                    tplist="$tp"
+             else
+                    tplist="$tplist,$tp"
+             fi
+        fi
+        ((n++))
+    done
+    echo $tplist
+}
+
+get_usage_and_exit(){
+    echo "usage: "
+    echo "FreeSurferPipelineBatch-long.sh [options]"
+    echo "options:"
+    echo "  --runlocal                      run locally [FALSE]"
+    exit -1
+}
```

Comment:
This function doesn't get used.  Batch scripts generally don't really need a usage statement, in my opinion, it is intended for the user to edit them.

Reply:
removed